ocs_ci/deployment/deployment.py

```diff
@@ -388,10 +393,8 @@ def deploy_ocs_via_operator(self):
                 'storage'
             ] = f"{device_size}Gi"
 
-        if self.platform.lower() == constants.VSPHERE_PLATFORM:
-            deviceset_data['dataPVCTemplate']['spec'][
-                'storageClassName'
-            ] = constants.DEFAULT_SC_VSPHERE
+        # set storage class to OCS default on current platform
+        deviceset_data['dataPVCTemplate']['spec']['storageClassName'] = self.DEFAULT_STORAGECLASS
```

Comment:
there is no any default storageclass for BM cluster how it getting handled

Reply:
@prsurve @petr-balogh How should a storage cluster CRD look like in case of baremetal? Should we:
- drop `storageClassName` from the spec section?
- set it to None?
- do something else?

Reply:
It should be LSO `localblock` in both cases.